source/agent/analytics/videoGstPipeline/GstInternalIn.cpp

```diff
@@ -64,7 +64,7 @@ void GstInternalIn::onFrame(const owt_base::Frame& frame)
         return;
     }
 
-    if(frame.additionalInfo.video.width == 1) {
+    if(frame->format > owt_base::FRAME_FORMAT_AV1 || frame->format < owt_base::FRAME_FORMAT_VP8) {
```

Comment:
Don't write condition like this. Use `isVideoFrame` function defined in MediaFramePipeline.h (https://github.com/open-webrtc-toolkit/owt-server/blob/master/source/core/owt_base/MediaFramePipeline.h#L209)